sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -170,6 +170,13 @@ class SessionCatalog(
     tableRelationCache.invalidate(key)
   }
 
+  /** Discards any cached table relation plans for the given table identifier. */
```

Comment:
nit. Shall we follow the comment style around this function? e.g. `This method discards ...`?

Reply:
Not sure. `SessionCatalog` has more method comments without any `This method ...`. It should obvious that method comment describes `This method ...`.

Reply:
I added "This method ..."